src/app/shop/pages/user-login-page/user-login-page.component.ts

```diff
@@ -13,27 +15,29 @@ import { AppState } from '../shared/store/app.states';
 })
 export class UserLoginPageComponent implements OnInit {
   form: FormGroup;
-  submitted = false;
+  getState: Observable<any>;
+  errorMessage: string | null;
 
   constructor(
     public auth: AuthCustomersService,
     private router: Router,
     private store: Store<AppState>
-  ) { }
+  ) {
+    this.getState = this.store.select(selectAuthState);
+  }
 
   ngOnInit() {
     this.form = new FormGroup({
       email: new FormControl(null, [Validators.required, Validators.email]),
       password: new FormControl(null, [Validators.required, Validators.minLength(6)]),
-    })
+    });
+
+    this.getState.subscribe((state) => {
+      this.errorMessage = state.errorMessage;
+    });
```

Comment:
If you need only `errorMessage` field from auth state please create specific selector for this